### ISSUE TITLE
Mac-OS result for aria-level missing

### ISSUE THREAD

**Issue by stevefaulkner** [TetraLogical/screen-reader-HTML-support#12]
https://tetralogical.github.io/screen-reader-HTML-support/VO-mac.html
https://tetralogical.github.io/screen-reader-HTML-support/lookup/lookup.html#aria-level

**Comment by stevefaulkner** [comment]
@lloydi did the tests i added them
